exchanges/poloniex/poloniex_futures.go

```
@@ -28,11 +28,7 @@ func (e *Exchange) GetAccountBalance(ctx context.Context) (*FuturesAccountBalanc
 // GetAccountBills retrieve the account’s bills.
 func (e *Exchange) GetAccountBills(ctx context.Context, startTime, endTime time.Time, offset, limit int64, direction, billType string) ([]BillDetail, error) {
 	params := url.Values{}
-	if !startTime.IsZero() && !endTime.IsZero() {
-		err := common.StartEndTimeCheck(startTime, endTime)
-		if err != nil {
-			return nil, err
-		}
+	if !startTime.IsZero() && !endTime.IsZero() && common.StartEndTimeCheck(startTime, endTime) != nil {
```

We shouldn't silence the error from a function, please revert back.